common/lib/capa/capa/templates/choicetext.html

```diff
@@ -24,7 +24,7 @@
                 % endif
             % endif
             >
-            <input class="ctinput" type="${input_type}" name="choiceinput_${id}" id="${choice_id}" value="${choice_id}"
+            <input class="ctinput" type="${input_type}" name="choiceinput_${id}" id="ctinput_${choice_id}" value="${choice_id}"
```

Comment:
What is the ID being used for? If nothing is referencing it, I would recommend deleting it and just using the class name for any necessary selectors.